docs/index.md

```diff
@@ -53,8 +53,10 @@ Pelorus works by integrating with your application platforms, developer tools, a
 -   # :simple-bitbucket:
 -   # :simple-gitea:
 -   # :simple-gitlab:
--   # :simple-azuredevops:
 -   # :simple-jirasoftware:
+-   # :simple-azuredevops:
+-   # :simple-trello:
```

Comment:
Should we add trello? (it is the same thing as jira?)

I would not add docker, because in the end we support only openshift (that uses the OCI tools)